src/main/java/appeng/client/gui/implementations/GuiCraftingStatus.java

```diff
@@ -249,6 +279,13 @@ private void updateCPUButtonText() {
         this.selectCPU.displayString = btnTextText;
     }
 
+    private void updateFollowButtonText() {
+        boolean isFollow = this.playersFollowingCurrentCraft.contains(this.mc.thePlayer.getCommandSenderName());
+
+        this.follow.displayString = isFollow ? GuiText.Follow.getLocal() : GuiText.Unfollow.getLocal();
+        this.follow.setTootipString(isFollow ? ButtonToolTips.Follow.getLocal() : ButtonToolTips.Unfollow.getLocal());
```

Comment:
Please reverse this logic. If the player is not yet following the current craft, the button should suggest following; otherwise, the button should be labeled "Unfollow".